.github/CODEOWNERS

```diff
@@ -32,3 +32,15 @@ tests/internal/remoteconfig         @DataDog/apm-core-python @DataDog/asm-python
 # Profiling
 ddtrace/profiling                   @DataDog/apm-core-python @DataDog/debugger-python
 tests/profiling                     @DataDog/apm-core-python @DataDog/debugger-python
+
+# Guild
+setup.py                            @DataDog/python-guild
+setup.cfg                           @DataDog/python-guild
+pyproject.toml                      @DataDog/python-guild
+.readthedocs.yml                    @DataDog/python-guild
+README.md                           @DataDog/python-guild
+mypy.ini                            @DataDog/python-guild
+.github/PULL_REQUEST_TEMPLATE.md    @DataDog/python-guild
+.github/ISSUE_TEMPLATE.md           @DataDog/python-guild
+ddtrace/internal/remoteconfig       @DataDog/python-guild
```

Comment:
this will overwrite the ASM/core ownership over remote config on L29

they are not additive, it'll take this one instead.

Reply:
I think that's what we want? The guild should own these.

Reply:
I think I agree, just confirming

for any changes to remoteconfig we require approvals from 2 members of the Python guild?


if we add a co-owner like core, then we could do 1 and 1 to widen the pool for approvals.

I'm not saying we have to make what I am suggesting, just talking out loud to confirm expected behavior.

Reply:
:+1: let's add core for the less important files so we can have better velocity